litmus-portal/frontend/src/components/Modal/FinishModal/index.tsx

```diff
@@ -10,18 +11,13 @@ function Icon() {
   return <img src="icons/finish.png" className={classes.mark} alt="mark" />;
 }
 
-const FinishModal = () => {
-  const classes = useStyles();
-
-  const [open, setOpen] = React.useState(false);
-
-  const handleOpen = () => {
-    setOpen(true);
-  };
+interface FinishModal {
```

Comment:
Would be better if named as FinishModalProps

Reply:
Sure.